test/src/sdk-harness/test_artifacts/reentrancy_attacker_contract/src/main.sw

```diff
@@ -1,20 +1,20 @@
 contract;
 
-use std::{chain::auth::*, context::call_frames::contract_id, contract_id::ContractId, result::*, revert::revert};
+use std::{chain::auth::*, context::call_frames::contract_id, contract_id::ContractId, result::*, revert::revert, identity::Identity};
 
 use reentrancy_target_abi::Target;
 use reentrancy_attacker_abi::Attacker;
 
 // Return the sender as a ContractId or panic:
-fn get_msg_sender_id_or_panic(result: Result<Sender, AuthError>) -> ContractId {
-    if let Result::Ok(s) = result {
-        if let Sender::ContractId(v) = s {
-            v
-        } else {
-            revert(0);
-        }
-    } else {
-        revert(0);
+fn get_msg_sender_id_or_panic(result: Result<Identity, AuthError>) -> ContractId {
+    match result {
+        Result::Ok(s) => {
+            match s {
+                Identity::ContractId(v) => {v},
```

Comment:
Pretty sure you do not need the brackets around the **v** etc., see it done in multiple places